qw11q/parameters.json

```diff
@@ -743,21 +789,21 @@
             "D1-D2": {
                 "CZ": [
                     {
-                        "duration": 70,
-                        "amplitude": 0.2,
-                        "shape": "GaussianSquare(5, 0.9)",
+                        "duration": 45,
+                        "amplitude": 0.1,
+                        "shape": "Google(0.55, 0)",
```

Comment:
What is this? xD

Reply:
I may have an idea about that, but it seems to be unsupported by both Qibolab 0.1 and 0.2

Reply:
Oh sorry I accidentally pushed some of my black magic that I am using to calibrate two qubit gates (without success). I'll remove them.